cluster-autoscaler/clusterstate/clusterstate.go

```diff
@@ -168,6 +172,11 @@ func NewClusterStateRegistry(cloudProvider cloudprovider.CloudProvider, config C
 		interrupt:                       make(chan struct{}),
 		scaleUpFailures:                 make(map[string][]ScaleUpFailure),
 		maxNodeProvisionTimeProvider:    maxNodeProvisionTimeProvider,
+		scaleDownDelayStatus: &node_processors.ScaleDownDelayStatus{
```

Comment:
I have moved `scaleDownDelayStatus` from `static_autoscaler` to clusterstate because adding it here makes it easier to update `scaleDownDelayStatus`  after scale-up [here](https://github.com/kubernetes/autoscaler/pull/5729/files#diff-95d98f54ac57fc3c60c23c1daca659d1fb752fc1822d32e42e12614b96331183R594). 

I might reconcile the code after I give a thought to https://github.com/kubernetes/autoscaler/pull/5729#discussion_r1186092461